### PR TITLE
PWN-7382 - Fix receive ethereum ui bugs

### PR DESCRIPTION
**Pull request by Davran331122**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7382

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/receive/eth/ReceiveEthereumFragment.kt [hunk 1]
```diff
@@ -62,6 +62,7 @@ class EthereumReceiveFragment :
                 showUiKitSnackBar(messageResId = R.string.receive_eth_address_copied)
             }
             financeBlockAddress.apply {
+                updateHorizontalPadding(start = 16, end = 8)
```
Comment (kabramovp2p) [inline app/src/main/java/org/p2p/wallet/receive/eth/ReceiveEthereumFragment.kt:65]: what the... it should not be like that

it should be a common design

Comment (Davran331122) [reply app/src/main/java/org/p2p/wallet/receive/eth/ReceiveEthereumFragment.kt:65]: Screenshot 2023-03-12 at 23 03 42

Comment (Davran331122) [reply app/src/main/java/org/p2p/wallet/receive/eth/ReceiveEthereumFragment.kt:65]: we have 24 px space on right side, that's why I add this method x D

Comment (NikolaiEmelianov) [reply app/src/main/java/org/p2p/wallet/receive/eth/ReceiveEthereumFragment.kt:65]: add `app:layout_goneMarginEnd="0dp"` to `@+id/leftSideView` in widget_finance_block.xml

File: ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/UiKitFinanceBlockView.kt [hunk 2]
```diff
@@ -44,6 +44,13 @@ class UiKitFinanceBlockView @JvmOverloads constructor(
         bindViewStyle(styleType)
     }
 
+    fun updateHorizontalPadding(start: Int, end: Int) {
```
Comment (NikolaiEmelianov) [inline ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/UiKitFinanceBlockView.kt:47]: not need if add layout_goneMarginEnd

**Review by NikolaiEmelianov — CHANGES_REQUESTED** [review]

**Review by gslevinkov — APPROVED** [review]

**Review by NikolaiEmelianov — APPROVED** [review]

**Pull request merged**